init.ts

```diff
@@ -81,7 +81,7 @@ async function createSuperuser() {
 		});
 }
 
-const app = express();
+export const app = express();
 app.enable('trust proxy');
 
 setup(app, {
```

Comment:
There should be a check in the tests that this has completed before starting to run since otherwise there'll be a race condition

Reply:
due to the way the tests are run, they only start after this promise resolves.